htdocs/js/jquery.dynamictable.js

```diff
@@ -98,6 +128,23 @@
                 checkOverflow(table.parentElement, rightLink, leftLink);
             });
 
+            if(colmNumber >= 0) {
```

Comment:
poorly chosen variable/function name.. there's no reason not to use the whole word "column"